### ISSUE TITLE
double console.log?

### ISSUE THREAD

**Issue by eitansuez** [bsletten/wasm_tdg#3]
https://github.com/bsletten/wasm_tdg/blob/aeb80fa9301706136363c580e82f1cc171e2dda9/ch03/imports.html#L25

>           console.log(instance.exports.log_how_old(2021, 2000));  

afaict log_how_old calls a function that calls console.log.  why wrap the call in yet another console.log ?

(hi brian).

**Comment by danielpacak** [comment]
Good catch. I run this example yesterday and confirmed that imported function does print logs, so wrapping doesn't really makes sene to me as well.
